CustomApps/lyrics-plus/index.js

```diff
@@ -220,6 +227,7 @@ class LyricsContainer extends react.Component {
     }
 
     async fetchLyrics(track, mode = -1) {
+        this.state.furigana, (this.state.romaji = null);
```

Comment:
What does this one-liner do exactly? Did you mean to assign both of them the value `null`?

Reply:
```suggestion
        this.state.furigana = this.state.romaji = this.state.hirgana = this.state.katakana = null;
```

Reply:
Yep that sounds about right, commit whenever you see fit.